_includes/navigation.html

```diff
@@ -4,70 +4,70 @@
     {% if site.about_enabled == true %}
       <li>
         <a href="{{ 'about' | relative_url }}" title="About me">
-          <span class="icon icon-about"></span>
+          <svg class="icon-about"><use xlink:href="{% asset_path about.svg %}#icon-about"></use></svg>
```

Comment:
Can you try changing the svg version to 2.0 and reversing the prop order?
If it doesn't work then sure, no point in breaking functionality.